angular-rails-app/spec/controllers/properties_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe PropertiesController do
+  describe "GET #index" do
+    it "returns a successful 200 response" do
+       get :index, format: :json
+      expect(response).to be_success
+    end
+
+    it "returns all properties" do
+      FactoryGirl.create_list(:property, 5)
+      get :index, format: :json
+      parsed_response = JSON.parse(response.body)
+      expect(parsed_response.length).to eq(5)
```

Comment:
Hmm, is this supposed to test that 5 properties will come back in json format? If so, you should also test that paging works. should be pretty easy to by using `FactoryGirl` to create 31 properties, and then ensure only 30 are returned.